src/Symfony/Component/Process/ProcessBuilder.php

```diff
@@ -11,13 +11,17 @@
 
 namespace Symfony\Component\Process;
 
+@trigger_error(sprintf('The %s class is deprecated since version 3.4 and will be removed in 4.0. Use the Process class instead.', __METHOD__), ProcessBuilder::class);
```

Comment:
The arguments at the end needs to be `, ProcessBuilder::class), E_USER_DEPRECATED);`